vxyowsup/whatsapp.py

```diff
@@ -60,11 +60,11 @@ def setup_transport(self):
 
     @defer.inlineCallbacks
     def teardown_transport(self):
-        print "Stopping client ..."
+        log.debug("Stopping client ...")
```

Comment:
I think the `teardown_transport` and `catch_exit` logs should use `log.info`. Apologies for not remembering these when I suggested using `log.debug`.